src/lib/matrix.js

```diff
@@ -103,3 +114,45 @@ exports.apply2DTransform2 = function(transform) {
         return at(xys.slice(0, 2)).concat(at(xys.slice(2, 4)));
     };
 };
+
+exports.convertCssMatrix = function(m) {
+    if(m) {
+        var len = m.length;
+        if(len === 16) {
+            // validate values
+            return [
+                m[0] || 1, m[1] || 0, m[2] || 0, m[3] || 0,
+                m[4] || 0, m[5] || 1, m[6] || 0, m[7] || 0,
+                m[8] || 0, m[9] || 0, m[10] || 1, m[11] || 0,
+                m[12] || 0, m[13] || 0, m[14] || 0, m[15] || 1
```

Comment:
Possibly we shouldn't try validating the matrix like this; which could sometimes be inaccurate.
I recall getting a `NaN` value in the first element of the matrix.
Instead of this, one should investigate why it happens at first place.

Reply:
Resolved by https://github.com/plotly/plotly.js/pull/5193/commits/28a917861d17ca923078b521612a08cc248bd23e.